capra-fagradar/package.json

```diff
@@ -16,7 +16,8 @@
 		"react": "^18.3.1",
 		"react-dom": "^18.3.1",
 		"react-router-dom": "^6.26.0",
-		"react-tooltip": "^5.28.0"
+		"react-tooltip": "^5.28.0",
+		"zustand": "^5.0.1"
```

Comment:
Ikke interesert i å introdusere zustand her. Zustand brukes som en "ekstern store" når en har data som skal endres. All data her er statisk, dette introduserer kun ekstra kompleksitet. Om du kun skal endre på areal for kvadranter så hold deg til endringer som kun har med dette å gjøre.

Reply:
Tanken med Zustand her er å holde styr på hvilken blip som er highlighted/valgt, ikke endre på noe data sådan. Den erstatter `const [currentBlip, setCurrentBlip] = useState()` og 3 nivåer med prop-drilling og utvider dette til å kunne sette en item man hovrer over for å highlighte tilhørende blip/`<li`> rundt radaren. (Hentet denne ideen fra fagradaren du viste frem forrige arbeidsmøte)

Man kan bruke Zustand til mer avanserte formål som du beskriver, men det funker ypperlig som et pent lite store for å slippe å skrive 30-40 linjer boilerplate for å lage en React Context/Provider. 😮‍💨 

Det kan kanskje være litt misvisende at denne endringen kommer med i en PR som heter "Endre på areal for hver kvadrant", men jeg kan oppdatere beskrivelsen til å inkludere dette?

Reply:
Jeg er enig i at zustand er bedre enn React Context/Provider, jeg bare syns det introduserer mer kompleksitet her hvor en useState hadde holdt lenge...